cmake/StopWatchConfig.cmake

```diff
@@ -0,0 +1,34 @@
+#.rst:
+# FindStopWatch
+# -------
+#
+# Find libStopWatch
+# StopWatch: wrapper around the std::chrono API for easy time expiration retrieval. 
+# AlarmClock: Low CPU clock cycle cost to retrieve if a time period has expired. 
+#
+# Result variables
+# ^^^^^^^^^^^^^^^^
+#
```

Comment:
For this and the other LROSS pull requests: 

I do not think the <repo name>Config.cmake are necessary. Further, they are hard to test also so I believe all of the special cmake config files should no be installed and in fact removed
(Murmurhash, FileIO, StopWatch, DeathKnell, Queuenado)